assets/src/edit-story/app/uploader/useUploader.js

```diff
@@ -162,13 +163,16 @@ function useUploader() {
 
       // Transcoding is enabled, let's give it a try!
       try {
+        setIsProcessing(true);
         // TODO: Only transcode & optimize video if needed (criteria TBD).
         const newFile = await transcodeVideo(file);
         trackTiming();
+        setIsProcessing(false);
         additionalData.media_source = 'video-optimization';
         return uploadMedia(newFile, additionalData);
       } catch (err) {
         trackTiming();
+        setIsProcessing(false);
```

Comment:
Just do this in a `finally`?

Reply:
We have the same problem that we had with the tracking. The return on 172.

Reply:
Dang, I looked for a `return` but I didn't see it 🤦

Reply:
Actually, the return is in the `try` block, not in the `catch` block. So doing the update in `finally` is totally possible without getting unexpected results.

Reply:
I get out a lint error, when I put it in.